agents/gce/fence_gce.py

```diff
@@ -230,7 +230,8 @@ def main():
 		if tuple(googleapiclient.__version__) < tuple("1.6.0"):
 			import oauth2client.client
 			credentials = oauth2client.client.GoogleCredentials.get_application_default()
-		conn = googleapiclient.discovery.build('compute', 'v1', credentials=credentials)
+		conn = googleapiclient.discovery.build(
+            'compute', 'v1', credentials=credentials, cache_discovery=False)
```

Comment:
Can you indent this a bit more, so it's clear it's part of the line above?

Reply:
Sure. For some reason `tabs` are used instead of `spaces` and my editor renders different... (that's the risk of using tabs...).